test/pacts/publishing_api/graphql_query_pact_test.rb

```diff
@@ -0,0 +1,42 @@
+require "test_helper"
+require "gds_api/publishing_api"
+
+describe "GdsApi::PublishingApi#graphql_query pact tests" do
+  include PactTest
+
+  let(:api_client) { GdsApi::PublishingApi.new(publishing_api_host) }
+
+  it "returns the response to the query" do
+    query = <<~QUERY
+      {
+        edition(basePath: "/my-document") {
+          ... on Edition {
+            title
+          }
+        }
+      }
+    QUERY
+
+    publishing_api
+      .given("a published content item exists with content_id: 19ad249e-7ac4-4aa4-8ab4-b6c5f381c043")
```

Comment:
**suggestion:** pertinent field

Per [my comment on the other PR](https://github.com/alphagov/publishing-api/pull/2950/files#r1822724361), it looks as though the base path is the meaningful field here for the Pact setup

Reply:
I've raised a PR to update the example and will also update it here.